### ISSUE TITLE
could not crop incomplete photos

### ISSUE THREAD

**Issue by royjor2016** [danmichaelo/croptool#134]
example: https://commons.wikimedia.org/wiki/File:Zentrale_Lotto_Rheinland-Pfalz.jpg

[Error] Command "/data/project/croptool/jpeg-9a/jpegtran -copy all -crop '3992x2475+0+0' '/mnt/nfs/labstore-secondary-tools-project/croptool/public_html/files/1b596d50a7942ce8de970db3087dee40c1e2bb33.jpg' > '/mnt/nfs/labstore-secondary-tools-project/croptool/public_html/files/1b596d50a7942ce8de970db3087dee40c1e2bb33_cropped.jpg' 2>&1" exited with code 2: 2

**Comment by danmichaelo** [comment]
Hm, I couldn't find an option to make jpegtran ignore errors, so lossless cropping might not be possible.

Did you try precise mode? Think that might work.

**Comment by danmichaelo** [comment]
Same as #131

**Comment by royjor2016** [comment]
Yes it works in Precise Mode. I've actually cropped incomplete ones before. I must have done it in precise too but I forgot. Still, is it possible to perform a lossless crop?

**Comment by danmichaelo** [comment]
Perhaps, but not to my knowledge.

**Comment by Jeff-Gdot** [comment]
Cropping https://commons.wikimedia.org/wiki/File:2019_Virginia_Avenue_train_tunnel_entrance.jpg worked in precise mode, perhaps you could generate a more user-friendly message when the user is trying to losslessly crop an incomplete file?

**Comment by danmichaelo** [comment]
Yeah, I also thought about that, the error message could definitely be improved. Perhaps something like "Precise crop failed for incomplete or invalid JPG file, please try precises mode."
